SamplesDashboard/SamplesDashboard/Extensions/AzureAdAuthenticationBuilderExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
```

Comment:
This looks like it is unused. Lets remove it.

Reply:
On it